### ISSUE TITLE
Missing CellDeps in the transactions displayed by Explorer

### ISSUE THREAD

**Issue by Flouse** [Magickbase/ckb-explorer-public-issues#531]
https://github.com/Magickbase/ckb-explorer-public-issues/issues/319 seems to be reappearing.

I can't reopen the issue, so I create a new one.

## Transaction
https://explorer.nervos.org/transaction/0x42fb5c1bba8db9b03f8468ade8c2f79cbe44f8a47b099b8f22742c070f0345ca

![image](https://github.com/Magickbase/ckb-explorer-public-issues/assets/1297478/389d557c-a666-450d-a998-75ec7b63cbe7)

**Comment by Keith-CY** [comment]
I have figured it out. Last week, due to the Omiga activity, there was a significant increase in the number of transactions in each block. Consequently, the handling of `cell deps` in each block also increased substantially, impacting the main process. To address this issue, we decided to set the processing of `cell deps` as an asynchronous worker, typically updating a few seconds later than the corresponding transactions (https://github.com/nervosnetwork/ckb-explorer/pull/1599).

The asynchronous processing approach was also applied to the data of COTA.

However, this morning (1/24), there was an error in the cota-aggregator, resulting in a large number of retry tasks. This, in turn, blocked the `cell deps` tasks. Now we switched the indexing service of COTA to https://cota.nervina.dev/mainnet-aggregator, and the accumulated tasks are now being executed one by one.
